news/6924.bugfix

```diff
@@ -0,0 +1,2 @@
+Fix an error when upgrade pip using ``pip.exe`` on Windows fails
```

Comment:
Maybe "Don't fail installation using pip.exe on Windows when pip wouldn't be upgraded."

Reply:
Thanks! Updated.